### ISSUE TITLE
import mesh_renderer error

### ISSUE THREAD

**Issue by xiaoshudi** [microsoft/Deep3DFaceReconstruction#89]
python=3.6 , tensorflow=1.12, cuda=9.0
I used 'pip install mesh_renderer' to install mesh_renderer.
Did anyone meet with this problem when import the mesh_renderer?

![image](https://user-images.githubusercontent.com/38290717/90329702-a9546880-dfd9-11ea-929b-d4d275cba842.png)

**Comment by YuDeng** [comment]
Can you try to use bazel to install tf mesh renderer? It seems that pip install is not stable for different environment.

**Comment by xiaoshudi** [comment]
I have tried to use bazel to install tf mesh renderer? but it always got some problems. firstly, I can't find the rasterize_trianales.so file. Then, I changed a library compiled by another person.  but also suffered from the same questions?
![image](https://user-images.githubusercontent.com/38290717/90393154-c3b64100-e0c2-11ea-98e4-222f18209dc2.png)

**Comment by YuDeng** [comment]
You might have to locate the rasterize_trianales.so file in your computer, and then modify the path in tf.load_op_library to your custom path.

**Comment by xiaoshudi** [comment]
Yeah, I did that as your suggestions, but also got the "undefined symbol:_ZN10tensorflow120pDefBuilder4AttrESs  " error.
Thank you very much for your kind help!

**Comment by Msj5** [comment]
> Yeah, I did that as your suggestions, but also got the "undefined symbol:_ZN10tensorflow120pDefBuilder4AttrESs " error.
> Thank you very much for your kind help!

Have you solved the "undefined symbol:_ZN10tensorflow120pDefBuilder4AttrESs " error. Thank.

**Comment by xiaoshudi** [comment]
> > Yeah, I did that as your suggestions, but also got the "undefined symbol:_ZN10tensorflow120pDefBuilder4AttrESs " error.
> > Thank you very much for your kind help!
> 
> Have you solved the "undefined symbol:_ZN10tensorflow120pDefBuilder4AttrESs " error. Thank.

I haven't solved!

**Comment by RainEggplant** [comment]
> 
> 
> > > Yeah, I did that as your suggestions, but also got the "undefined symbol:_ZN10tensorflow120pDefBuilder4AttrESs " error.
> > > Thank you very much for your kind help!
> > 
> > 
> > Have you solved the "undefined symbol:_ZN10tensorflow120pDefBuilder4AttrESs " error. Thank.
> 
> I haven't solved!

When compiling `rasterize_triangles_kernel.so`, change g++ param `-D_GLIBCXX_USE_CXX11_ABI=0` to `-D_GLIBCXX_USE_CXX11_ABI=1`. This solves my problem.

**Comment by xinxxx0124** [comment]
> > > > Yeah, I did that as your suggestions, but also got the "undefined symbol:_ZN10tensorflow120pDefBuilder4AttrESs " error.
> > > > Thank you very much for your kind help!
> > > 
> > > 
> > > Have you solved the "undefined symbol:_ZN10tensorflow120pDefBuilder4AttrESs " error. Thank.
> > 
> > 
> > I haven't solved!
> 
> When compiling `rasterize_triangles_kernel.so`, change g++ param `-D_GLIBCXX_USE_CXX11_ABI=0` to `-D_GLIBCXX_USE_CXX11_ABI=1`. This solves my problem.

I change it but it makes another error， then I change tensorflow version to 1.15.2 and now it is ok.  And the output .so file is in tf_mesh_render/bazel-bin/mesh_render/kernels/  .

**Comment by jiangtaowudi** [comment]
> python=3.6 , tensorflow=1.12, cuda=9.0 我使用“pip install mesh_renderer”安装了 mesh_renderer。 有人在导入 mesh_renderer 时遇到过这个问题吗？
> 
> ![image](https://user-images.githubusercontent.com/38290717/90329702-a9546880-dfd9-11ea-929b-d4d275cba842.png)

**Comment by jiangtaowudi** [comment]
> > python=3.6 , tensorflow=1.12, cuda=9.0 我使用“pip install mesh_renderer”安装了mesh_renderer。有人在导入mesh_renderer时遇到这个问题吗？
> > ![image](https://user-images.githubusercontent.com/38290717/90329702-a9546880-dfd9-11ea-929b-d4d275cba842.png)

**Comment by jiangtaowudi** [comment]
可以加个好友交流一下吗？ 我也是卡在了这里

**Comment by jiangtaowudi** [comment]
> python=3.6 , tensorflow=1.12, cuda=9.0 I used 'pip install mesh_renderer' to install mesh_renderer. Did anyone meet with this problem when import the mesh_renderer?
> 
> ![image](https://user-images.githubusercontent.com/38290717/90329702-a9546880-dfd9-11ea-929b-d4d275cba842.png)

请问解决了吗博主
